### PR TITLE
Pets part 2. I'm still stupid edition

### PR DESCRIPTION
**Pull request by NigglyWiggly**
Somehow forgot Jonas and Sasha, so here they are now!

### CODE REVIEW

**Comment by ComicIronic** [comment]
This is why you learn to rebase, dummy.

File: code/modules/mob/living/simple_animal/friendly/corgi.dm [hunk 1]
```diff
@@ -529,4 +528,28 @@
 				emote("yaps happily")
 		else
 			if(M && stat != DEAD) // Same check here, even though emote checks it as well (poor form to check it only in the help case)
-				emote("growls")
\ No newline at end of file
+				emote("growls")
+
+
+//Sasha isn't even a corgi you dummy!
+/mob/living/simple_animal/corgi/Sasha
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm]: >UPPERCASE OBJECT NAMES

NO

File: code/modules/mob/living/simple_animal/hostile/giant_spider/spiderling.dm [hunk 2]
```diff
@@ -133,4 +133,13 @@
 	//ventcrawl!
 	if(!v.welded)
 		entry_vent = v
-		Goto(get_turf(v),move_to_delay)
\ No newline at end of file
+		Goto(get_turf(v),move_to_delay)
+
+//Virologist's little friend!
+/mob/living/simple_animal/hostile/giant_spider/spiderling/Jonas
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/hostile/giant_spider/spiderling.dm]: >UPPERCASE TYPE PATHS

NO

**Comment by d3athrow** [comment]
Goof's right change them to lowercase.

**Comment by Iamgoofball** [comment]
uppercase type paths are literally hitler

**Pull request merged**